app/org/sagebionetworks/bridge/models/studies/StudyParticipant.java

```diff
@@ -45,7 +45,9 @@ public Boolean getNotifyByEmail() {
         return (emptyString == null) ? null : Boolean.valueOf(emptyString);
     }
     public void setNotifyByEmail(Boolean notifyByEmail) {
-        put(UserProfile.NOTIFY_BY_EMAIL_FIELD, notifyByEmail.toString());
+        if (notifyByEmail != null) {
+            put(UserProfile.NOTIFY_BY_EMAIL_FIELD, notifyByEmail.toString());    
+        }
```

Comment:
Why would you get a call to this method with a null Boolean? Additionally, should it PUT "false" if the variable is null rather than do nothing?